src/EntityComponentManager.cc

```diff
@@ -772,6 +772,15 @@ void EntityComponentManagerPrivate::SetRemovedComponentsMsgs(Entity &_entity,
 
   // Find the entity in the message
   auto entIter = _msg.mutable_entities()->find(_entity);
+  // Add the entity to the message, if not already added
```

Comment:
can you add a small comment briefly explaining what you mentioned in the PR description?

Reply:
Done, in https://github.com/ignitionrobotics/ign-gazebo/pull/495/commits/8285e15ccfc2c82fa23f21a7eedec72e9d994f2f